packages/myst-to-jats/src/frontmatter.ts

```diff
@@ -166,7 +167,7 @@ export function getArticleAuthors(frontmatter: ProjectFrontmatter): Element[] {
             attributes: {
               vocab: 'CRediT',
               'vocab-identifier': 'http://credit.niso.org/',
-              'vocab-term': `${role}`,
+              'vocab-term': escapeForXML(role),
```

Comment:
@fwkoch is there a better place to do this so we don't run into it again?

Reply:
Hmm, seems like the only other place, besides at this granular level would be at the xml serialization level. Looking at the docs, there doesn't seem to be any escaping built into `xml-js` but there are some hooks for processing functions... https://github.com/nashwaan/xml-js#options-for-custom-processing-functions - so maybe we could, like, escape _all_ attributes with an `attributeValueFn`...?

Reply:
Not sure if there are implications of being too heavy handed there...? I guess we would mostly just need to be careful about not escaping twice (though our escape function is already idempotent I think, so maybe that's not a concern at all).

Reply:
Seems to work ok with our existing test suite at least - see what you think, I just made a pr that merges into this branch: https://github.com/executablebooks/mystmd/pull/649

Reply:
Nice, this looks great, thank you.